src/write_gcm.c

```diff
@@ -3775,6 +4000,54 @@ static void wg_log_json_message(const wg_context_t *ctx, const char *fmt, ...) {
   fflush(ctx->json_log_file);
 }
 
+static char *wg_get_instance_id_from_monitored_resource(
+    monitored_resource_t *resource) {
+  if (resource == NULL) {
+    ERROR(
+        "write_gcm: wg_get_instance_id_from_monitored_resource resource empty");
+    return NULL;
+  }
+  for (int i = 0; i < resource->num_labels; i++) {
+    if(strcmp(*(resource->keys + i), "instance_id") == 0) {
```

Comment:
`resource->keys[i]` is the same as `*(resource->keys + i)`, but more readable. Ditto for `resource->values[i]` vs. `*(resource->values + i)`.